instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -427,12 +427,19 @@ def traced_execution(
             if args and self._commenter_enabled:
                 try:
                     args_list = list(args)
+                    if hasattr(self._connect_module, "__libpq_version__"):
+                        libpq_version = self._connect_module.__libpq_version__
+                    else:
+                        libpq_version = (
+                            self._connect_module.pq.__build_version__
+                        )
```

Comment:
This is expected to raise an exception every time its called from outside psycopg, right? Would it make sense to move this logic to the psycopg instrumentation and pass a callback for the commenter?

Reply:
It may be out of the scope to do this refactoring, so I am happy to merge as is.

Reply:
Would prefer this as I'm not familiar with the codebase yet! Would rather just make psycopg work for now.